spec/translation_quality/checks/blacklisted_terms.yaml

```diff
@@ -0,0 +1,124 @@
+definitions:
+  account_check_types_BLACKLISTED_TERMS_settings_effective:
+    get:
+      description: >-
+        Get checktype BLACKLISTED_TERMS settings for certain account.
```

Comment:
Change to: "Get BLACKLISTED_TERMS settings for the check type for certain accounts.